README.md

```diff
@@ -145,6 +146,21 @@ To clone it into a folder with a different name than the default repository name
 $ git clone [url] name-of-new-folder
 ```
 
+### I set the wrong remote repository.
+
+There are a few possible problems here:
+
+
+If you cloned the wrong respistory simply delete the directory created after running git clone and clone the correct repository
+
+
+If you set the wrong repository as the origin of an existing local repository, change the url of your origin by running 
+
+```sh
+$ git remote set-url origin [url of the actual repo]
+```
+For more see [this SO top](https://stackoverflow.com/questions/2432764/how-to-change-the-uri-url-for-a-remote-git-repository#2432799)
```

Comment:
Remove. Let's not link out.

Reply:
The readme litterally states to link to SO and sources.

Reply:
Huh, I forgot that. :D Good catch. Keep this in.